# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/fs-uae/hotfix-xatom.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

libfsemu/src/ml/x11.c: In function 'set_window_icon':
libfsemu/src/ml/x11.c:112:56: error: 'XA_CARDINAL' undeclared (first use in this function)
  112 |     XChangeProperty(g_display, g_window, _NET_WM_ICON, XA_CARDINAL, 32,

--- fs-uae-3.1.66/libfsemu/src/ml/x11.c.vanilla	2024-08-30 12:26:18.411000000 +0100
+++ fs-uae-3.1.66/libfsemu/src/ml/x11.c	2024-08-30 12:26:29.344000000 +0100
@@ -26,6 +26,7 @@
 
 #include <X11/Xlib.h>
 #include <X11/keysym.h>
+#include <X11/Xatom.h>
 #ifdef USE_X11_XTEST
 #include <X11/extensions/XTest.h>
 #endif
